pkg/manifests/assets/gateway-api/aggregated-cluster-roles/admin-cluster-role.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:gateway-api:aggregate-to-admin
```

Comment:
This is a fairly generic, non-OpenShift-specific name for a cluster-scoped resource.  What happens if the cluster-admin is already using this name for a clusterrole?

Reply:
Yeah, this is just draft cruft I was looking for a better name I think I found one based on looking at other ones other operators used :+1: